README.md

```diff
@@ -1,63 +1,107 @@
+		Open Source Marketing Automation Software.
+    <br />
+    <a href="https://mautic.org"><strong>mautic.org 🡥</strong></a>
+    <br />
+    <br />
+    <a href="https://m.mautic.org/mautic-start-your-trial">Request a trial</a> · <a href="https://www.mautic.org/download">Self-host/download</a> · <a href="https://www.mautic.org/community">Community channels</a>
+</p>
+
+<p align="center">
+  <a href="https://codecov.io/gh/mautic/mautic"><img alt="Codecov" src="https://codecov.io/gh/mautic/mautic/branch/features/graph/badge.svg"></a>
+  <a href="#contributors-#contributors-"><img alt="All Contributors" src="https://img.shields.io/badge/all_contributors-141-orange.svg?style=flat-square"></a>
+</p>
+<br/>
+<br/>
+
+🗒️ About Mautic
 ============
-Mautic is the world’s largest open source marketing automation project. With over 200,000 organisations using Mautic and over 1,000 community volunteers, we empower businesses by making it easy to manage their marketing across a range of channels. Stay up to date about initiatives, releases and strategy via our [blog][mautic-blog].
+Mautic is the world’s largest open source marketing automation project. With over 200,000 organisations using Mautic and over 1,000 community volunteers, we empower businesses by making it easy to manage their marketing across a range of channels. 
+
+### ✨ Benefits over regular marketing automation tools
+
+- 🌐 It's open-source - use it for free, no strings attached
+- 🏠 Don't rely on third-party SaaS, run your own marketing automation
+- 🗄️ Use your own database and own your data
+- 🔓 No vendor lock-in - switch or modify at will
+- 🔧 Create any integration you need
+- ⚡ Build stronger campaigns, without feature limitations
+- 🛠️ Both frontend and backend are 100% customizable
+- 📡 Multi-channel marketing without extra costs
+- 📊 Unlimited segmentation and automation
+
+Stay up to date about initiatives, releases and strategy via our [blog][mautic-blog].
 
 Marketing automation has historically been difficult to implement within organisations. The Mautic Community is an example of open source at its best, offering great software and a vibrant and caring community in which to learn and share knowledge.
 
 Open source means more than open code. Open source provides equality for all and a chance for everyone to improve.
 
+
 ![Mautic](.github/readme_image.png "Mautic Open Source Marketing Automation")
 
-Get Involved
+Get Involved Quickly
 =============
-Before we tell you how to install and use Mautic, we like to shamelessly plug our awesome user and developer communities! Users, start [here][get-involved] for inspiration, or follow us on Twitter [@MauticCommunity][twitter] or Facebook [@MauticCommunity][facebook]. Once you’re familiar with using the software, maybe you will share your wisdom with others in our [Slack][slack] channel.
+Calling all devs, testers and tech writers! Get started quickly and show us what you’re made of. Technical contributions are also welcome.
```

Comment:
I'm not sure why we say 'technical contributions also welcome' when we've just asked for devs and technical writers - are we implying they are not technical?

Perhaps we could re-word it or remove it?